lib/suite.js

```diff
@@ -152,6 +154,24 @@ Suite.prototype.bail = function(bail){
 };
 
 /**
+  * Set randomization `enabled`.
+  *
+  * Do NOT use `this.enableRandomize()` in your test; use `describe.noRandom`
+  * instead.
+  *
+  * @param {Boolean} enabled
+  * @return {Suite|Boolean} self or enabled
+  * @api private
+  */
+
+Suite.prototype.enableRandomize = function(enabled){
+  if (arguments.length === 0) return this._enableRandomize;
```

Comment:
Nvm, without arguments it's used as a getter... I think it would be bad to see `.enableRandomize()` around being used as a getter – not logical at all.